### PR TITLE
Added test cases Framework_Stability_001 WebKit_Memory_001

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Added below test cases for this Pull request : 
Framework_Stability_001.test 
WebKit_Memory_001.test

### CODE REVIEW

File: src/framework/Framework_Stability_001.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+  getControllerPluginData,
+} from '../commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller Robustness test',
+  description: 'Tests start/stop functionality of the controller module and repeats 100 times',
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/framework/Framework_Stability_001.test.js]: No need

File: src/framework/Framework_Stability_001.test.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  steps: [
+    {
+      description: 'Get Controller data and validate whether it is Object or not',
+      test: getControllerPluginData,
```
Comment (wouterlucas) [inline src/framework/Framework_Stability_001.test.js:13]: should be a jsonrpc api request

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/framework/Framework_Stability_001.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Stability_001.test.js]: Updated as per the code review comment by removing the teardown method

File: src/framework/Framework_Stability_001.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Stability_001.test.js:13]: Actual getControllerPluginData method is a jsonrpc request. So this comment is marked as resolved

**Comment by sandeep-vedam** [comment]
@wouterlucas , Comments are addressed for this PR. Please check and take appropriate action if it's ok.

**Pull request merged**